comp/core/tagger/taggerimpl/server/server.go

```diff
@@ -27,9 +27,21 @@ const (
 	streamKeepAliveInterval = 9 * time.Minute
 )
 
+// streamIDManager is used to generate unique ID's for incoming streaming requests
+// This unique ID is used to subscribe to the tagger
+// TODO: remove this struct when the protobuf of the stream request is updated to use filters.
+type streamIDManager int
+
+func (s *streamIDManager) getNextUniqueID() string {
+	id := fmt.Sprintf("stream-client-%d", *s)
+	*s++
+	return id
+}
+
```

Comment:
This is just a temporary solution. 

Ideally, we will modify the protobuf message to include full data needed to perform the subscription to the tagger, including the subscription id. 

I preferred to keep modifying the protobuf to a separate PR to make it easier for review.